### ISSUE TITLE
Padrino 0.14.1 fails to start w/o haml

### ISSUE THREAD

**Issue by pdlug** [padrino/padrino-framework#2136]
Padrino 0.14.1 fails to start if haml is not in the bundle, this worked in 0.14.0.2 (probably related to the change to support hamlit?).

```
LoadError: cannot load such file -- haml
/usr/src/app/vendor/ruby/2.4.0/gems/tilt-2.0.7/lib/tilt/haml.rb:2:in `require'
/usr/src/app/vendor/ruby/2.4.0/gems/tilt-2.0.7/lib/tilt/haml.rb:2:in `<top (required)>'
/usr/src/app/vendor/ruby/2.4.0/gems/tilt-2.0.7/lib/tilt/mapping.rb:243:in `require'
/usr/src/app/vendor/ruby/2.4.0/gems/tilt-2.0.7/lib/tilt/mapping.rb:243:in `block in lazy_load'
/usr/src/app/vendor/ruby/2.4.0/gems/tilt-2.0.7/lib/tilt/mapping.rb:241:in `each'
/usr/src/app/vendor/ruby/2.4.0/gems/tilt-2.0.7/lib/tilt/mapping.rb:241:in `lazy_load'
/usr/src/app/vendor/ruby/2.4.0/gems/tilt-2.0.7/lib/tilt/mapping.rb:216:in `lookup'
/usr/src/app/vendor/ruby/2.4.0/gems/tilt-2.0.7/lib/tilt/mapping.rb:154:in `[]'
/usr/src/app/vendor/ruby/2.4.0/gems/tilt-2.0.7/lib/tilt.rb:53:in `template_for'
/usr/src/app/vendor/ruby/2.4.0/gems/padrino-helpers-0.14.1/lib/padrino-helpers/output_helpers/haml_handler.rb:5:in `<module:OutputHelpers>'
/usr/src/app/vendor/ruby/2.4.0/gems/padrino-helpers-0.14.1/lib/padrino-helpers/output_helpers/haml_handler.rb:3:in `<module:Helpers>'
/usr/src/app/vendor/ruby/2.4.0/gems/padrino-helpers-0.14.1/lib/padrino-helpers/output_helpers/haml_handler.rb:2:in `<module:Padrino>'
/usr/src/app/vendor/ruby/2.4.0/gems/padrino-helpers-0.14.1/lib/padrino-helpers/output_helpers/haml_handler.rb:1:in `<top (required)>'
/usr/src/app/vendor/ruby/2.4.0/gems/padrino-support-0.14.1/lib/padrino-support/file_set.rb:23:in `require'
/usr/src/app/vendor/ruby/2.4.0/gems/padrino-support-0.14.1/lib/padrino-support/file_set.rb:23:in `block in glob_require'
/usr/src/app/vendor/ruby/2.4.0/gems/padrino-support-0.14.1/lib/padrino-support/file_set.rb:14:in `block in glob'
/usr/src/app/vendor/ruby/2.4.0/gems/padrino-support-0.14.1/lib/padrino-support/file_set.rb:14:in `each'
/usr/src/app/vendor/ruby/2.4.0/gems/padrino-support-0.14.1/lib/padrino-support/file_set.rb:14:in `glob'
/usr/src/app/vendor/ruby/2.4.0/gems/padrino-support-0.14.1/lib/padrino-support/file_set.rb:23:in `glob_require'
/usr/src/app/vendor/ruby/2.4.0/gems/padrino-helpers-0.14.1/lib/padrino-helpers.rb:9:in `<top (required)>'
/usr/src/app/vendor/ruby/2.4.0/gems/padrino-0.14.1/lib/padrino.rb:4:in `require'
/usr/src/app/vendor/ruby/2.4.0/gems/padrino-0.14.1/lib/padrino.rb:4:in `<top (required)>'
```

**Comment by vuhailuyen1991** [comment]
same issue here when upgrading to 0.14.1.  
But i think they have fixed in the latest version 0.14.1.1, please upgrade again and verify

**Comment by pdlug** [comment]
Confirmed, fixed in 0.14.1.1.
